### ISSUE TITLE
API to list zones

### ISSUE THREAD

**Issue by kno10** [thecodingmachine/workadventure#1448]
It would be good to have an API to iterate over all zones, in order to attach listeners.

**Comment by moufmouf** [comment]
Hey @kno10,

I think you can already get that by writing a utility function wrapping a call to `WA.room.getTiledMap()`.

I tend to restrict to a minimum the number of functions that make it to the API.
On the other end, I started an "extra" utility package that anyone can use and that adds a bunch of utility functions. See https://github.com/workadventure/scripting-api-extra/ (I'm planning to communicate on this widely in the coming weeks).

So basically, a function to iterate on zones would be a good candidate for "scripting-api-extra" (rather than the API itself)

**Comment by moufmouf** [comment]
Oh, ok!
Some code from the -extra package requires features that are not yet released (we plan to release this week!)
